### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by treufuss**
Fix background of QR code used for login with smartphone

### CODE REVIEW

**Review by AndrewDaGuy — CHANGES_REQUESTED** [review]
Gusted likes non-hard coded colours, perhaps set it to `${white}`?

**Comment by treufuss** [comment]
> Gusted likes non-hard coded colours, perhaps set it to `${white}`?

I think it has to be hard coded in this case. The (black) QR code is a png with a transparent background, so i want to set the background of the surrounding div to white in order to make the QR code always visible.

QRExample

**Comment by AndrewDaGuy** [comment]
> I think it has to be hard coded in this case. The (black) QR code is a png with a transparent background, so i want to set the background of the surrounding div to white in order to make the QR code always visible.
> 
> QRExample

Maybe just invert the QR code?

**Comment by Gusted** [comment]
> @treufuss Maybe just invert the QR code?

Andrew, I just like to suggest to read this sentence once again. With a special note how QR code scanning works ;)

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -274,6 +274,15 @@ INVERT
 
 ================================
 
+access.ing.de
+
+CSS
+.tile__info {
+  background-color: #fff !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:281]: 4 Indents & the hard coded color is good as it could otherwise break the QR code implementation

**Review by AndrewDaGuy — APPROVED** [review]
Andrew Approved :+1:, just finish the changes Gusted requested ;]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:281]:
```suggestion
    background-color: #fff !important;
```

**Review by Gusted — APPROVED** [review]

**Pull request merged**